mitosheet/mitosheet/public/v3/sheet_functions/number_functions.py

```diff
@@ -55,10 +82,182 @@ def AVG(*argv: NumberInputType) -> NumberFunctionReturnType:
+        ],
+        "syntax": "FLOAT(string_series)",
+        "syntax_elements": [{
+                "element": "string_series",
+                "description": "The series or string to convert to a float."
+            }
+        ]
+    }
+    """
+    if isinstance(series, int) or isinstance(series, float):
+        return float(series)
+    
+    return series
+
+@cast_values_in_arg_to_type('series', 'int')
+@handle_sheet_function_errors
+def INT(series: IntRestrictedInputType) -> IntFunctionReturnType:
+    """
+    {
+        "function": "INT",
+        "description": "Converts a string series to a int series. Any values that fail to convert will return 0.",
+        "search_terms": ["number", "to integer"],
+        "examples": [
+            "=INT(Prices_string)",
+            "=INT('123')"
+        ],
+        "syntax": "INT(string_series)",
+        "syntax_elements": [{
+                "element": "string_series",
+                "description": "The series or string to convert to a int."
+            }
+        ]
+    }
+    """
+    if isinstance(series, int):
+        return series
+    
+    return series
+    
+
+
+@cast_values_in_arg_to_type('series', 'number')
+@handle_sheet_function_errors
+def KURT(series: NumberRestrictedInputType) -> NumberFunctionReturnType:
+    """
+    {
+        "function": "KURT",
+        "description": "Computes the unbiased kurtosis, a measure of tailedness, of a series, excluding missing values.",
+        "search_terms": ["kurtosis"],
+        "examples": [
+            "=KURT(A)",
+            "=KURT(A * B)"
+        ],
+        "syntax": "KURT(series)",
+        "syntax_elements": [{
+                "element": "series",
+                "description": "The series to calculate the unbiased kurtosis of."
+            }
+        ]
+    }
+    """
+    if isinstance(series, int) or isinstance(series, float):
+        return 0
+        
+    return series.kurt() # type: ignore
+
+
+@cast_values_in_arg_to_type('series', 'number')
+@cast_values_in_arg_to_type('base', 'number')
+@handle_sheet_function_errors
+def LOG(series: NumberRestrictedInputType, base: Optional[NumberRestrictedInputType]=None) -> NumberFunctionReturnType:
+    """
+    {
+        "function": "LOG",
+        "description": "Calculates the logarithm of the passed series with an optional base.",
+        "search_terms": ["log", "logarithm", "natural log"],
+        "examples": [
+            "LOG(e) = 1",
+            "LOG(100, 10) = 2"
+        ],
+        "syntax": "LOG(series, [base])",
+        "syntax_elements": [{
+                "element": "series",
+                "description": "The series to take the logarithm of."
+            },
+            {
+                "element": "base [OPTIONAL]",
+                "description": "The base of the logarithm to use. Defaults to the natural logarithm if no base is passed."
+            }
+        ]
+    }
+    """
+    
+    if base is None:
+        base = math.e
+
```

Comment:
Excel uses 10 as the base value

<img width="770" alt="Screenshot 2023-03-28 at 12 24 02 PM" src="https://user-images.githubusercontent.com/18709905/228305844-79b75582-5c7a-4028-92e9-e7e40821d3f3.png">